client/app/app.less

```diff
@@ -1,2 +1,14 @@
-
 @import 'css/source.less';
+
+@nprogress-background-color: @oui-color-california;
+
+/* Make clicks pass-through */
+#nprogress {
```

Comment:
I would recommend to use a more precise selector instead of using the `!important`.

I mean that we can set a default class on the `body` or the `html` element (in `client/app/index.html` file) and then refer to it.

```html
<!-- client/app/index.html --->
<body class="ovh-manager-dedicated-app">
```

```less
// client/app/app.less
.ovh-manager-dedicated-app {
  #nprogress {
    .bar {
      background: @nprogress-background-color;
    }

    .peg {
      box-shadow:
        0 0 10px @nprogress-background-color,
        0 0 5px @nprogress-background-color;
    }
  }
}
```

Reply:
Everyone is using `!important` in their CSS :(

We should ban it altogether, except for utility classes, and we shouldn't have to create utility classes as `Tailwind` exists 😄

Reply:
@antleblanc @FredericEspiau First idea was to allow theming but as the style is injected through the Javascript file, I'm having trouble to find a way to do it 
I'll go with @antleblanc solution